src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationService.scala

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import javax.ws.rs.Path
+
+import akka.actor.{Props, Actor}
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model.{WorkspaceName, Destination, MethodConfigurationCopy}
+import org.broadinstitute.dsde.firecloud.{FireCloudConfig, HttpClient}
+import org.slf4j.LoggerFactory
+import spray.client.pipelining._
+import spray.routing._
+import spray.httpx.SprayJsonSupport._
+
+class MethodConfigurationServiceActor extends Actor with MethodConfigurationService {
+  def actorRefFactory = context
+  def receive = runRoute(routes)
+}
+
+@Api(value = "/workspaces/{workspaceNamespace}/{workspaceName}/method_configs",
+  description = "Method Configuration Services",
+  produces = "application/json")
+trait MethodConfigurationService extends HttpService with FireCloudDirectives {
+
+  private final val ApiPrefix = "workspaces"
+  lazy val routes = copyMethodRepositoryConfigurationRoute
+  lazy val log = LoggerFactory.getLogger(getClass)
+
+  @Path(value = "/copyFromMethodRepo")
+  @ApiOperation(
+    value = "copy a method repository configuration to a workspace",
+    nickname = "copyMethodRepositoryConfigurationToWorkspace",
+    httpMethod = "POST",
+    notes = "Copy a Method Repository Configuration into a workspace")
+  @ApiImplicitParams(Array(
+    new ApiImplicitParam(name = "workspaceNamespace", required = true, dataType = "string", paramType = "path", value = "Workspace Namespace"),
+    new ApiImplicitParam(name = "workspaceName", required = true, dataType = "string", paramType = "path", value = "Workspace Name"),
+    new ApiImplicitParam(name = "configurationNamespace", required = true, dataType = "string", paramType = "form", value = "Configuration Namespace"),
```

Comment:
During standup last week, we discussed the two approaches of either sending a full json object to orchestration or sending the data values as form fields. Everyone, at the time, agreed with the simpler approach of using form fields in conjunction with path parameters to then construct the object that gets passed to rawls.